starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,38 @@
 // Names and Input
+var hacker1 = "Montse";
+console.log("The driver's name is " + hacker1);
 
+var hacker2 = prompt("Enter navigator's name");
+console.log("The navigator's name is " + hacker2)
 
 //Conditionals
+if (hacker1.length > hacker2.length) {
+  console.log("The driver has the longest name, it has " + hacker1.length + "characters.")
+} else if (hacker2.length > hacker1.length) {
+  console.log("The navigator has the longest name, it has " + hacker2.length + "characters.")
+} else {
+  console.log("Wow, you both got equally long names, " + hacker1.length + "characters!!!!!!!")
+};
+
+var name = "";
+var uppercase = hacker1.toUpperCase();
+var name2 = "";
+for (var i=0; i<hacker1.length; i++) {
+  name += uppercase[i]+ " ";
+}
+console.log(name);
+var value = hacker1.length -1;
```

Comment:
it's ok, but you don't need to create a variable for that. It will not modify its value if you modify the hacker1 variable later in your program.